shared/templates/accounts_password/oval.template

```diff
@@ -11,9 +11,29 @@
       <criteria operator="OR">
         <criterion comment="pwquality.conf" test_ref="test_password_pam_pwquality_{{{ VARIABLE }}}" />
       </criteria>
+      {{% if "ol" in product %}}
+      <criterion comment="{{{ VARIABLE }}} is not overwritten in system-auth"
+      test_ref="test_password_pam_pwquality_{{{ VARIABLE }}}_not_overwritten"/>
+      {{% endif %}}
     </criteria>
   </definition>
 
+  {{% if "ol" in product %}}
+  <ind:textfilecontent54_test check="all" check_existence="none_exist"
+  comment="check the configuration of /etc/pam.d/system-auth doens't override pwquality.conf"
```

Comment:
nitpick: could you adjust the indentation please?

Reply:
sorry what's the issue with indentation?